src/js/modules/supervisors/FileManager.ts

```diff
@@ -30,6 +30,9 @@ import {
  * this class is instanced
  */
 class FileManager {
+  private watcher: Inotify;
+  private removeWatcher: Inotify;
```

Comment:
Do you need two inotify instances to do this? Can you just do something like watcher.add_watch(this.activeDir) ?

Reply:
Umm no, this libray `Inotify` just allow adds watches by folder or path, but I can try to change it in the next PR where I change this library, what do you think? @graphcareful

Reply:
That is adding a watch by a path. If your going to change this anyway , i guess it doesn't matter